source/jobs/prerequisites/oracle.txt

```diff
@@ -90,11 +97,11 @@ Steps
                   GRANT SELECT ON V_$ARCHIVE_DEST_STATUS TO <user>;
                   GRANT SELECT ON V_$TRANSACTION TO <user>;
                   GRANT SELECT ON V_$MYSTAT TO <user>;
-                  GRANT SELECT ON V_$STATNAME TO <user>; 
+                  GRANT SELECT ON V_$STATNAME TO <user>;
 
          .. step:: Turn on archive logging
             
-            a. To see if archive logging is already enabled, run the following query:
+            a. To examine if archive logging is already enabled, run the following query:
```

Comment:
same as with multi-tenancy check, in my personal opinion, "to examine" is a very strong / fancy word here for a mere "true / false" check